### DISCUSSION TITLE
HM_BUILD_ID error during update of Openradioss version

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#2991] [Q&A]
Hello everybody, 

I'm trying to update the version of OpenRadioss that I'm currently using with an updated one from the Release repository. 

Unfortunately, both with source code installation and with the compiled versione, I get the same error: starter_win64.exe - Entry point not found The entry point HM_BUILD_ID of the procedure could not be found in the dynamic link library C:\OpenRadioss\exec\starter_win64.exe

How could I solve this problem?

![image](https://github.com/user-attachments/assets/aef3ae09-9752-45d6-abc0-fab57cc76acd)

**Comment by sebastienVilleneuve** [comment]
Hi @FGarelli 

Could you check if 'PATH' is pointing to the updated repository ?

cf https://github.com/OpenRadioss/OpenRadioss/blob/main/INSTALL.md 
![image](https://github.com/user-attachments/assets/51a1cc27-5a29-46b2-a1a1-ec402086bf2a)

Thanks,
Sébastien

**Reply by FGarelli** [reply]
Hello @sebastienVilleneuve,

Yes, I launch simulations from cmd with the correct variable sets. Here what I'm setting:

set OPENRADIOSS_PATH=C:\OpenRadioss,%OPENRADIOSS_PATH%
set RAD_CFG_PATH=%OPENRADIOSS_PATH%\hm_cfg_files
set RAD_H3D_PATH=%OPENRADIOSS_PATH%\extlib\h3d\lib\win64
set KMP_STACKSIZE=400m
set PATH=%OPENRADIOSS_PATH%\extlib\hm_reader\win64;%PATH%
set PATH=%OPENRADIOSS_PATH%\exec;%PATH%
set OMP_NUM_THREADS=1
set PATH=C:\Program Files (x86)\Intel\oneAPI\compiler\2023.1.0\windows\redist\intel64_win\compiler;%PATH% 
call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat" intel64 vs2022

Federico

**Comment by sebastienVilleneuve** [comment]
Hi @FGarelli ,

Can you recheck , using the last stable version , checking if the path is pointing to the 'up to date' library ?
( but maybe this pb is already solved on you side ?)

Thanks,
Sébastien

**Reply by FGarelli** [reply]
Hello @sebastienVilleneuve ,
I tried to install the last stable version (source code and compiled) but I get the same error. Still didn't solve the problem.
